adr/recipe.py

```diff
@@ -101,10 +146,23 @@ def run_recipe(recipe, args, config):
     """
     modname = '.recipes.{}'.format(recipe)
     mod = importlib.import_module(modname, package='adr')
-    try:
-        output = mod.run(args, config)
-    except MissingDataError:
-        return "ActiveData didn\'t return any data."
+
+    if recipe == 'config_durations':
+        # get value of query parameters and post-processing parameters
+        query_args, _ = StructureRecipeParser(mod.get_query_args()).parse_known_args(args)
+        processing_args, _ = StructureRecipeParser(mod.get_pp_args()).parse_known_args(args)
```

Comment:
Instead of defining `get_query_args` and `get_pp_args`, just access the global directly here.

Reply:
I removed get_query_args (because we can extract them automatically from *.query). The `get_pp_args` is still kept (rename to `get_run_definition`) because it belongs to each recipe so it is hard to set into global variable.